submissions/WEremite/html-css-popup/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <title>HTML-CSS-POPUP</title>
+</head>
+<body>
+    <nav class="menu">

```

Comment:
Need to correct validator errors.
See link [here](https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fweremite.github.io%2Fhtml-css-popup%2F#l92c29)

Reply:
Done.